fedmsg/meta/base.py

```diff
@@ -154,7 +154,9 @@ def handle_msg(self, msg, **config):
             return match.groups()[-1] or ""
 
     def title(self, msg, **config):
-        return '.'.join(msg['topic'].split('.')[3:])
+        if msg['topic'][0].isalpha():
```

Comment:
It makes my head spin trying to imagine all the different combinations of dots..  :dizzy_face: